test/e2e_node/runtime_conformance_test.go

```diff
@@ -28,111 +32,319 @@ import (
 )
 
 const (
-	retryTimeout = time.Minute * 4
+	retryTimeout = time.Minute * 5
 	pollInterval = time.Second * 5
 )
 
-var _ = Describe("Container Runtime Conformance Test", func() {
+type testStatus struct {
+	Name             string
+	RestartPolicy    api.RestartPolicy
+	Phase            api.PodPhase
+	State            ContainerState
+	RestartCountOper string
+	RestartCount     int32
+	Ready            bool
+}
+
+var _ = Describe("Container runtime Conformance Test", func() {
 	var cl *client.Client
 
 	BeforeEach(func() {
 		// Setup the apiserver client
 		cl = client.NewOrDie(&restclient.Config{Host: *apiServerAddress})
 	})
 
-	Describe("container conformance blackbox test", func() {
-		Context("when running a container that terminates", func() {
-			var containerCase ConformanceContainer
+	Describe("container runtime conformance blackbox test", func() {
+		namespace := "runtime-conformance"
+		Context("when start a container that exits successfully", func() {
+			var testCContainers []ConformanceContainer
 
 			BeforeEach(func() {
-				containerCase = ConformanceContainer{
-					Container: api.Container{
-						Image:           "gcr.io/google_containers/busybox",
-						Name:            "busybox",
-						Command:         []string{"sh", "-c", "env"},
-						ImagePullPolicy: api.PullIfNotPresent,
+				testCContainers = []ConformanceContainer{}
+			})
+
+			It("it should run with the expected status [Conformance]", func() {
```

Comment:
It also checks other status. I'll change it to 'expected restart count and status'.